frost-p256/src/lib.rs

```diff
@@ -225,6 +225,14 @@ impl Ciphersuite for P256Sha256 {
             m,
         ))
     }
+
+    /// HID for FROST(P-256, SHA-256)
+    fn HID(m: &[u8]) -> Option<<<Self::Group as Group>::Field as Field>::Scalar> {
```

Comment:
This function is different than in the other ciphersuites but does the same thing. I think it would be good to have them all be written the same way. (Unless I'm misunderstaind something)